pkg/integration/tests/commit/commit.go

```diff
@@ -33,5 +34,18 @@ var Commit = NewIntegrationTest(NewIntegrationTestArgs{
 			Lines(
 				Contains(commitMessage),
 			)
+		t.Views().Files().
+			IsFocused().
+			PressPrimaryAction().
+			Press(keys.Files.CommitChanges).
+			Press(keys.Universal.PrevItem)
```

Comment:
rather than have this press command here I'd have a method on the commit message panel driver (which we should rename to commit summary panel driver) called 'SelectPreviousMessage' and 'SelectNextMessage' which internally will use `Press(keys.Universal.PrevItem)` and `Press(keys.Universal.NextItem)`